se.manyver.Manyverse.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: se.manyver.Manyverse
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: manyverse
+separate-locales: false
+rename-desktop-file: manyverse.desktop
+rename-icon: manyverse
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --allow=bluetooth
+  # full filesystem access is needed in order for the user to be able to browse and upload files from their filesystem
+  - --filesystem=home:ro
+  - --filesystem=xdg-config/manyverse
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.portal.Background
+  - --own-name=org.kde.*
```

Comment:
this is a very broad dbus own.

Reply:
That's also good point, I might not even need an own name. How do I know exactly if I need a own name?